.github/workflows/centos_configure.sh

```diff
@@ -45,5 +45,5 @@ ${KRATOS_CMAKE_OPTIONS_FLAGS} \
 -DCMAKE_CXX_FLAGS="${KRATOS_CMAKE_CXX_FLAGS} -O0 -Wall" \
 -DCMAKE_UNITY_BUILD=ON                              \
 
-# Buid
+# Build
```

Comment:
Line 8 can be changed to:
You can find a list `with` all the `compilation` options in INSTALL.md or here: